### ISSUE TITLE
Header Button Lacks Icon on Mobile Web

### ISSUE THREAD

**Issue by ckaznable** [mattermost/mattermost-plugin-calls#780]
The primary reason for this issue is that the `registerCallButtonAction` method dispatches both `CallButton` and `MobileChannelHeaderButton` simultaneously but does not pass an icon, resulting in this behavior.

Additionally, I have also opened a related issue in the Mattermost repository.
https://github.com/mattermost/mattermost/issues/27359

screenshot:
![image](https://github.com/mattermost/mattermost-plugin-calls/assets/18453951/b96dd61d-46f6-499c-826d-ea90f534643b)

**Comment by streamer45** [comment]
@ckaznable I don't believe web view should be a supported way to use this functionality from mobile. It may work, but it wasn't part of the design.

@cpoile @matthewbirtch Thoughts?

**Comment by cpoile** [comment]
I agree.. Perhaps we should not register it on mobile web?

**Comment by ckaznable** [comment]
I just checked community.mattermost.com and did not encounter the same issue. Perhaps it's just a configuration issue on my end?

**Comment by streamer45** [comment]
@ckaznable Could you share a screenshot of how it would look like when working? I don't remember having a button to start calls in that view.

**Comment by ckaznable** [comment]
when I press this button, it triggers the plugin's behavior. After disabling the plugin, the button no longer appears, so I'm sure it's the button from this plugin.

![image](https://github.com/mattermost/mattermost-plugin-calls/assets/18453951/75df2a14-37d1-4639-b815-acd0fc10d4a8)

**Comment by matthewbirtch** [comment]
> @ckaznable I don't believe web view should be a supported way to use this functionality from mobile. It may work, but it wasn't part of the design.
> 
> @cpoile @matthewbirtch Thoughts?

Agreed, plugin actions were not intended to show in the header on mobile web view.

**Comment by ckaznable** [comment]
When I use the desktop version and reduce the screen width, this situation also occurs. I think this is not limited to mobile web.

**Comment by streamer45** [comment]
Yes, that's exactly the same view, regardless of the device.

**Comment by streamer45** [comment]
Closing this as fixed. The "Start call" button will be available through the channel header menu dropdown.

![Image](https://github.com/user-attachments/assets/3eb1e676-6217-4b71-88e0-baa8b32f510a)
